test/geometry/nerf/test_nerf_model.py

```diff
@@ -0,0 +1,33 @@
+import torch
+
+from kornia.geometry.nerf.nerf_model import MLP, NerfModel
+
+
+class TestNerfModel:
+    def test_mlp(self, device, dtype):
+        d_input = 63  # Input dimension after encoding
+        num_hidden = 256
+        mlp = MLP(d_input, num_units=2, num_unit_layers=4, num_hidden=num_hidden)
```

Comment:
```suggestion
        mlp = MLP(d_input, num_units=2, num_unit_layers=4, num_hidden=num_hidden)
        mlp = mlp.to(device, dtype)
```

Reply:
Doesn't this done recursively when I move the NeRF model to device in: nerf_solver.py:93?